Tribler/Test/Community/gigachannel/test_community.py

```diff
@@ -0,0 +1,320 @@
+from twisted.internet.defer import Deferred, inlineCallbacks
+
+from Tribler.community.gigachannel.community import ChannelDownloadCache, GigaChannelCommunity
+from Tribler.pyipv8.ipv8.peer import Peer
+from Tribler.pyipv8.ipv8.test.base import TestBase
+
+
+class MockChannel(object):
```

Comment:
Some of these mock classes are helpful in other tests. Could you extract them to a more generic location so we can reuse them across different tests?